en/development/testing.rst

```diff
@@ -463,11 +463,23 @@ database schema as well::
     // Run the Documents migrations on the test_docs connection.
     $migrator->run(['plugin' => 'Documents', 'connection' => 'test_docs']);
 
+If you need to run multiple sets of migrations, those can be run as follows::
+
+    // Run migrations for plugin Contacts on
+    // the ``test`` connection, and Documents on the ``test_docs`` connection
+    $migrator->runMany([
+        ['plugin' => 'Contacts'],
+        ['plugin' => 'Documents', 'connection' => 'test_docs']
```

Comment:
Does this only run the plugins? Is the app fixtures automatically run? It's kind of unclear.

Reply:
Use `[]` to run app migrations.
Like this

```php
    // Run migrations for plugin Contacts on
    // the ``test`` connection, and Documents on the ``test_docs`` connection
    $migrator->runMany([
        ['plugin' => 'Contacts'],
        ['plugin' => 'Documents', 'connection' => 'test_docs'],
        [], // app migrations
    ]);
```

Reply:
I've updated the example, let me know if it is clearer now.

Reply:
Yep thank you! One of the issues we were having was how to run both app and plugin but all the examples had it just for the app or plugins and not both. Thank you!